assets/js/blocks/mini-cart/mini-cart-contents/index.tsx

```diff
@@ -41,6 +41,10 @@ const settings: BlockConfiguration = {
 			link: true,
 		},
 		lock: false,
+		__experimentalBorder: {
```

Comment:
Is it correct that we enable this feature in WC Core too? I'm asking because it is an experimental API

Reply:
Yes, you are right, it shouldn't. I'll fix it. Thanks :)

Reply:
Pushed the fix, can you have another look? @gigitux